modules/gdscript/tests/scripts/runtime/features/does_not_override_temp_values.gd

```diff
@@ -0,0 +1,17 @@
+# https://github.com/godotengine/godot/issues/71177
+
+func test():
+	builtin_method()
+	builtin_method_static()
+	print("done")
+
+func builtin_method():
+	var pba := PackedByteArray()
+	@warning_ignore(return_value_discarded)
+	pba.resize(1) # Built-in validated.
+
+
+func builtin_method_static():
+	var _pba := PackedByteArray()
```

Comment:
Duplicated line?

Reply:
No, this is relevant for the test. This creates a PackedByteArray in the stack so the test makes sure that value is not being corrupted. I guess a comment would be good here.